src/server/api/routes/questions.router.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const questionsController = require('../controllers/questions.controller');
+
```

Comment:
@dpfernandes The above three lines are also repeated across PR's

Reply:
I did these overlap for purpose. We have two issues for api/questions get and post. But both issues deal with same table. So, in my opinion there should be a single route for get and post api for questions table. After merging both PR in the develop branch we will get only one route api/questions for get and post requests. Does it make sense?